pkg/services/ngalert/models/alert_rule.go

```diff
@@ -240,6 +240,8 @@ type AlertRule struct {
 	PanelID         *int64  `xorm:"panel_id"`
 	RuleGroup       string
 	RuleGroupIndex  int `xorm:"rule_group_idx"`
+	Record          string
```

Comment:
This also begs the question - we should probably either rename `AlertRule` to just `Rule`, since it might not actually be an alerting rule any more... or have separated domain model for alert rules vs recording rules that are combined in the API/rule table.

I'm considering both not in scope for this PR and leaving them for the future, as they're deep refactor type changes.